generators/server/templates/_build.gradle

```diff
@@ -453,12 +449,23 @@ clean {
     delete "target"
 }
 
+task copyDockerFiles(type: Copy) {
```

Comment:
> I can't import the tasks in another gradle file.

@atomfrede @oehme it works if you define the `buildscript` section on the file as above